src/lib/Support/Generator.cpp

```diff
@@ -114,5 +101,37 @@ buildOneString(
     }
 }
 
+Error
+Generator::
+buildOne(
+    std::ostream& os,
+    Corpus const& corpus) const
+{
+    return Error("buildOne() not implemented for this generator type");
```

Comment:
All generators implement this function. It's the most basic function they have. There's no point to make not abstract just to return a runtime error saying it's not implemented. That's the exact problem the abstract version solves in compile time.